CMakeLists.txt

```diff
@@ -19,14 +19,18 @@ add_custom_target(add-copyright
   COMMENT "Add copyright")
 
 if(MSVC)
-  # Link executables statically by replacing /MD with /MT everywhere.
-  foreach(flag_var
-      CMAKE_CXX_FLAGS CMAKE_CXX_FLAGS_DEBUG CMAKE_CXX_FLAGS_RELEASE
-      CMAKE_CXX_FLAGS_MINSIZEREL CMAKE_CXX_FLAGS_RELWITHDEBINFO)
-    if(${flag_var} MATCHES "/MD")
-      string(REGEX REPLACE "/MD" "/MT" ${flag_var} "${${flag_var}}")
-    endif(${flag_var} MATCHES "/MD")
-  endforeach(flag_var)
+  option(ENABLE_SHARED_CRT "Use the shared CRT instead of the static CRT"
```

Comment:
Suggest we use a SHADERC_ prefix so someone could control this even when Shaderc is embedded in a larger project.